base/compiler/ssair/inlining.jl

```diff
@@ -1287,6 +1287,16 @@ function handle_const_opaque_closure_call!(
     return nothing
 end
 
+function inline_const_if_inlineable!(ir, idx)
```

Comment:
```suggestion
function inline_const_if_inlineable!(ir::IRCode, idx::Int)
```

Reply:
Perhaps ideally this would take an Instruction as an argument directly? I was hoping to move in that direction, but didn't want to cause excessive churn by changing existing cases